extension-base/index.js

```diff
@@ -0,0 +1,53 @@
+// var wrapper = require('./ctpAzureWrapper.js');
+// module.exports = wrapper.ctpAzureWrapper(addInsurance);
+
+var wrapper = require('./ctpGcfWrapper.js');
+exports.handler = wrapper.ctpGcfWrapper(addInsurance);
+
+function addInsurance(input, ctpResponse, log) {
+  // Use an ID from your project!
+  var taxCategoryId = "af6532f2-2f74-4e0d-867f-cc9f6d0b7c5a";
+
+  var cart = input.resource.obj;
+  // If the cart contains any line item that is worth more than $500,
+  // mandatory insurance needs to be added.
+  var itemRequiresInsurance = cart.lineItems.find( (lineItem) => {
+      return lineItem.totalPrice.centAmount > 50000;
+  });
+  var insuranceItem = cart.customLineItems.find( (customLineItem) => {
+      return customLineItem.slug == "mandatory-insurance";
+  });
+
+  var cartRequiresInsurance = itemRequiresInsurance != undefined;

```

Comment:
if your intention is to use a boolean value in the end based on having a key available within a list, the following would be cooler:

```js
// `some` => if any of the iteration returns true
// `every` => if every iteration returns true
const cartRequiresInsurance = cart.lineItems.some(
  lineItem => lineItem.centAmount > 5000
)
```

---

[ref for every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)
[ref for some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)